torchao/sparsity/prototype/superblock/utils.py

```diff
@@ -12,6 +12,25 @@
 import torch
 import torch.distributed as dist
 
+### IMAGENET UTILS
+@torch.inference_mode
+def benchmark_inference(warmup, iters, f, *args, **kwargs):
```

Comment:
why not use https://github.com/pytorch/ao/blob/5998389cb7304ed40f09251ed75e50632d00c142/torchao/utils.py#L51? we can add a warmup as well

Reply:
It was giving me different timings from my function. I'll try and debug the delta later, if I can find out whats causes it I'll switch over.